### ISSUE TITLE
QA Webpack

### ISSUE THREAD

**Issue by LeoMcA** [mozilla/sumo#976]
Regression tests for Webpack on https://support.allizom.org/

**Comment by mirunacurtean** [comment]
I have been executing the test cases from the [SUMO End to End test run](https://testrail.stage.mozaws.net/index.php?/runs/view/51514&group_by=cases:section_id&group_order=asc&group_id=125889), but so far the only new issue is the All products filter (from search results sorting) not being highlighted. 
![All products filter by product is not highlighted when first search is perfoemed](https://user-images.githubusercontent.com/53306802/146942180-333b66dc-f5b1-4ee4-a8bb-ade9ab44c143.gif)

I've added and updated with with inserted media (images, video, internal links) questions, articles, community posts with no new issues (still cannot upload gif or insert media taken with a camera right in the process of updating the post).


Found a couple of issues that already occur in production and will add separate tickets for those. 
display issue in Sent messages when the username is long
![display issue in Sent messages when the username is long](https://user-images.githubusercontent.com/53306802/146942848-a451a48a-6d5b-46f5-bd4b-cb6d876498b1.PNG)

avatar picture is not updated in sumo until after sign out sign in
![avatar picture is not updated in sumo until after sign out sign in](https://user-images.githubusercontent.com/53306802/146942857-818ef13b-3986-4369-bad5-c684dd57d9a8.gif)
